modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -67,6 +68,7 @@ def __init__(
         index_cols=None,
         uses_rowid=False,
         force_execution_mode=None,
+        is_default_frame=False,
```

Comment:
I think `is_default_frame` doesn't really reflect the meaning of the attribute. Probably `has_unsupported_data`?

Reply:
renamed